src/applications/simple-forms/mock-simple-forms-patterns/pages/mockArrayMultiPageBuilderPages.js

```diff
@@ -0,0 +1,133 @@
+import {
```

Comment:
instead of several different pages, I moved all the consumer code to one page, so it's easier to copy pasta when you need to use this